src/components/header/ProfileThreadJankOverview.js

```diff
@@ -2,6 +2,8 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
+// @flow
+
```

Comment:
Flow was happy with no changes. I suppose it could infer everything it wanted. Or do we want to add explicit types anyway, for instance for the parameters to `connect`?